oelint_adv/__main__.py

```diff
@@ -140,6 +145,13 @@ def group_files(files):
 
     return res.values()
 
+def print_ruleset(args):
+    rules = [x for x in load_rules(args,
+        add_rules=args.addrules, add_dirs=args.customrules)]
+    ruleset = {r.ID: r.Severity for r in rules}
+    json.dump(ruleset, sys.stdout, indent=2)
```

Comment:
Why not use `print(json.dumps(ruleset, indent=2, sort_keys=True) + "\n")`? I don't like to see empty print statements in the code, as they often are falsely flagged by linting tools

Reply:
Right, I was just used to using the stream interface, to not allocate a large string for the json object.  But this is not that much output so it does not matter.  I changed it.